src/main/java/nl/novi/techiteasy1121/Dtos/TelevisionInputDto.java

```diff
@@ -1,164 +1,177 @@
-package nl.novi.techiteasy1121.Dtos;
+package nl.novi.techiteasy1121.dtos;
 
-// Let er op dat je hier uit het "jakarta" package importeert
 import jakarta.validation.constraints.*;
 
-// Deze klasse wordt gebruikt voor je Post en Put methodes, dus daar waar je een Television als body mee geeft in Postman.
-
 public class TelevisionInputDto {
```

Comment:
Zorg ervoor dat je de validatie annotaties in je DTO's goed toepast. Dit helpt om ervoor te zorgen dat de invoer van de gebruiker voldoet aan de vereisten voordat deze naar de service wordt gestuurd.